go-controller/pkg/ovn/egressfirewall.go

```diff
@@ -387,9 +390,15 @@ func (oc *DefaultNetworkController) addEgressFirewallRules(ef *egressFirewall, p
 			action = nbdb.ACLActionDrop
 		}
 		if len(rule.to.nodeAddrs) > 0 {
-			for _, addr := range sets.List(rule.to.nodeAddrs) {
-				// ideally we don't care about sorting this list, but this is being done to ensure Unit Test consistency
-				// and its not like this nodeAddrs can be super large per node EFW rule to cause scale issues
+			// sort node ips to ensure the same order when no changes are present
+			// this ensure ACL recalculation won't happen just because of the order changes
+			allIPs := []string{}
+			for _, nodeIPs := range rule.to.nodeAddrs {
+				allIPs = append(allIPs, nodeIPs...)
+			}
+			slices.Sort(allIPs)
```

Comment:
maybe
`sets.New(maps.Values(rule.to.nodeAddrs)).List()`

Reply:
that is quite cool, but I don't think it will work, as every map value is a list itself, so `maps.Values(rule.to.nodeAddrs)` is [][]string, so you need something like 
`sets.New(slices.Concat(maps.Values(rule.to.nodeAddrs)...)...).UnsortedList()`
instead, which I think at this point is more difficult to read that just a loop, wdyt?

Reply:
yeah 2 nodes with the same ip is kinda unexpected, but not that bad from the ACL POV